### PR TITLE
New queries

### PR DESCRIPTION
**Pull request by Xterionix**
Also made it alphabetical

### CODE REVIEW

File: src/MolangData/MolangFunction.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@ import { Types } from "bc-minecraft-bedrock-types";
  *
  */
 export interface MolangFunction extends Types.Identifiable, Types.Documentated {
+
+  id : string;
+
+  documentation: string;
```
Comment (DaanV2) [inline src/MolangData/MolangFunction.ts:10]: These are already in the Types.Identifiable and Types.Documentated 🤔

**Review by DaanV2 — CHANGES_REQUESTED** [review]

File: src/MolangData/MolangFunction.ts (hunk 1 above)
Comment (Xterionix) [reply src/MolangData/MolangFunction.ts]: I didn't see them in the autocomplete so I added them

**Review by DaanV2 — APPROVED** [review]

File: src/MolangData/General.ts [hunk 2]
```diff
@@ -606,6 +726,10 @@ export namespace General {
       id: "is_angry",
       documentation: "Returns 1.0 if the entity is angry, else it returns 0.0.",
     },
+    {
+      id: "is_attached",
+      documentation: "Returns 1.0 if the entity is attached to another entity (such as being held or worn), else it will return 0.0. Available only with resource packs." 
+    }
```
Comment (DaanV2) [inline src/MolangData/General.ts:732]:
```suggestion
    },
```

**Review by DaanV2 — APPROVED** [review]

File: src/MolangData/MolangFunction.ts (hunk 1 above)
Comment (DaanV2) [reply src/MolangData/MolangFunction.ts]: Nw

**Pull request merged**